roles/pulp/templates/requirements.in.j2

```diff
@@ -1,5 +1,11 @@
 pulpcore=={{ pulp_version }}
-{% for plugin, value in pulp_install_plugins.items() %}
+{% for plugin, value in pulp_install_plugins_normalized.items() %}
 {{ plugin }}{% if value['version'] is defined and value['version']|length %}=={{ value['version'] }}{% endif %}
 
 {% endfor %}
+# Any plugins listed below were already installed but not specified in
+# pulp_install_plugins
+{% for plugin in pip_pkgs.packages[pulp_install_dir + '/bin/pip'].keys() %}
+{% if "pulp-" in plugin and plugin not in pulp_install_plugins_normalized.keys() %}{{ plugin }} {% endif %}
```

Comment:
`galaxy-ng` would not fall in this rule, the only way I see for solving it is having a default var:
```yaml
supported_plugins:
  - pulp-file
  - galaxy-ng
  - ...
  - pulp-ansible
```

```
{% if plugin in supported_plugins and plugin not in pulp_install_plugins_normalized.keys() %}{{ plugin }} {% endif %}
```

Reply:
I'm developing another commit now. It adds to roles/pulp/defaults/main.yml a list called "pulp_irregularly_named_plugins", with currently 1 element "galaxy_ng". The template will check if the plugin either begins with "pulp-" or has that irregular name.

I wonder whether it should be a public variable (advertised in the pulp role's README.md) or not?

Reply:
I just force pushed that change.

Reply:
I like this approach, for now, I believe we don't need to advertise it as `galaxy-ng` is the only case so far.